architecture/service_catalog/ansible_service_broker.adoc

```diff
@@ -0,0 +1,129 @@
+[[arch-ansible-service-broker]]
+= Ansible Service Broker
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+ifdef::openshift-origin,openshift-enterprise[]
+To opt-in during installation or upgrade, see Installation & Configuration (TODO link).
+endif::[]
+====
+
+The Ansible service broker (ASB) is an implementation of the OSB API that
+manages applications defined by xref:service-catalog-apb[_Ansible playbook bundles (APBs)_]. APBs provide a new method for defining and distributing
+container applications in {product-title}, consisting of a bundle of Ansible
+playbooks built into a container image with an Ansible runtime. APBs leverage
+Ansible to create a standard mechanism for automating complex deployments.
+
+The design of the ASB follows this basic workflow:
+
+. A user requests list of available applications from the service catalog using
+the {product-title} web console.
+. The service catalog requests the ASB for available applications.
+. The ASB communicates with a defined container registry to learn which APBs are
+available.
+. The user issues a request to provision a specific APB.
+. The provision request makes its way to the ASB, which fulfills the user's
+request by invoking the provision method on the APB.
+
+image::asb-arch.png["Ansible Service Broker Architecture"]
+
+[[service-catalog-apb]]
+== Ansible Playbook Bundles
+
+An Ansible playbook bundle (APB) is a lightweight application definition for
+defining, allowing you to leverage existing investment in Ansible roles and
```

Comment:
"An Ansible playbook bundle (APB) is a lightweight application definition for defining, allowing you to leverage existing investment in Ansible roles and"

The "...for defining" part read a little strange to me, perhaps just remove that for:

"An Ansible playbook bundle (APB) is a lightweight application definition, allowing you to leverage existing investment in Ansible roles and"